SECURITY.md

```diff
@@ -0,0 +1,62 @@
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+<!-- BEGIN DellEMC SECURITY.MD V0.0.1 BLOCK -->
+
+## Security
+
+DellEMC takes the security of our software products and services seriously, which includes all source code repositories managed through our GitHub organizations, which include [Pravega](https://github.com/pravega/pravega).
```

Comment:
This and other statements in this document have ingredients of a proprietary corporate policy and references a company. It should be agnostic to a particular company, and shouldn't be referencing Dell EMC in my view.